client-js/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.0] - 2023-04-03
+
+### Changed
+
+- Added `casper-js-sdk` and `@make-software/ces-js-parser` as a peer dependencies to prevent conflicts between multiple versions of dependencies
```

Comment:
```suggestion
- Added `casper-js-sdk` and `@make-software/ces-js-parser` as peer dependencies. This will prevent conflicts between multiple versions of dependencies.
```